### DISCUSSION TITLE
How to override extension's `createKeymap`?

### DISCUSSION THREAD

**Discussion by drl990114** [remirror/remirror#2152] [Q&A] [answered]
I want to override the shortcut keys of the extension. Based on some guidelines in the documentation, I tried the following code, but it did not take effect. I still executed the keymap binding function of the plugin, so I would like to ask how to do it?

```js
const hooks = [
  () => {
           useKeymaps(
            {
              'Mod-Shift-s': () => {
                console.log('Mod-Shift-s')
                return false
              },
            },
            ExtensionPriority.Critical,
          )
  },
];

const Editor = () => {
  const { manager } = useRemirror({ extensions: () => [new BoldExtension()] });

  return <Remirror manager={manager} hooks={hooks} />;
};
```

**Comment by ocavue** [comment, marked as answer]
You can return `true` in your code snippet to stop other key bindings from handling it.

**Reply by drl990114** [reply]
Thank you for your timely response！

Yes, initially I returned `true`, but it didn't work, so I tried `false` again to return the value, but no changes

**Reply by drl990114** [reply]
I have created a new warehouse and it cannot be reproduced. It should be a problem with my code. Thank you for your reply
